### DISCUSSION TITLE
Stacked Bar Graphs

### DISCUSSION THREAD

**Discussion by magicroomy** [GSI-CS-CO/chart-fx#353] [Ideas]
Hi, I could not find any example for Stacked Bar Graphs. Did I just miss them or is there not support ?

**Comment by wirew0rm** [comment]
There is no flag that you can just enable, but there is a stacked histogram plot in the `HistogramRendererSample`. It works by plotting the different datasets on top of each other, while using a `MathDataSet` to add the value of all the remaining datasets. (restriction: your data has to be all positive.)

You could make a lightweight renderer wrapper, so you do not have to manually wrap the datasets, see `CustomFragmentedRendererSample`, for en example of such a wrapper.

This also shows why it is nontrivial to add this as a renderer feature, because the renderers at the moment work on a per dataset basis. Additionally it would have to handle differently sampled data and a lot of other corner cases gracefully.
